Pseudocode/Queue/First Non Repeating in Stream/SourceCode.tex

```diff
@@ -0,0 +1,59 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\newcommand{\IsNotEmpty}{\textbf{ is not empty, }}
```

Comment:
You only need a space before the text if you plan to use it inside Math mode. If you are using it in normal mode, then it creates extra space. Hence, the recommended way is to always use it inside Math mode (See the gist).